cloudformation/convert_types.go

```diff
@@ -8,12 +8,12 @@ func String(v string) *string {
 }
 
 // Strings returns a pointer to a slice of string pointers.
-func Strings(v... string) *[]string {
+func Strings(v ...string) []string {
```

Comment:
Let's keep this as it was before (so it works as the documentation intended), and replace all the `cloudformation.Strings` usage with real literal strings `[]string{...}`.

I know it's a lot of find/replace, but let me know if I can help with this!